### PR TITLE
Issue #SH-502 fix: Auto play course content

### PR DESCRIPTION
**Pull request by NavKumarV**

### CODE REVIEW

File: src/app/collection-detail-etb/collection-detail-etb.page.ts [hunk 1]
```diff
@@ -1354,54 +1352,24 @@ export class CollectionDetailEtbPage implements OnInit {
   }
 
   playContent(event) {
-    this.headerService.hideHeader();
+    const telemetryDetails = {
```
Comment (swayangjit) [inline src/app/collection-detail-etb/collection-detail-etb.page.ts:1355]: Why page Id is COURSE_DETAILS in Collection page ?

File: src/app/curriculum-courses/chapter-details/chapter-details.page.ts [hunk 2]
```diff
@@ -445,8 +448,11 @@ export class ChapterDetailsPage implements OnInit, OnDestroy {
     );
     if (this.childContents.length && !this.isBatchNotStarted) {
       const firstChild = this.loadFirstChildren(this.chapter);
-      this.navigateToChildrenDetailsPage(firstChild, 1);
-    } else {
+      const telemetryDetails = {
+        pageId: PageId.COURSE_DETAIL,
```
Comment (swayangjit) [inline src/app/curriculum-courses/chapter-details/chapter-details.page.ts:452]: PageId is incorrect

File: src/app/curriculum-courses/chapter-details/chapter-details.page.ts [hunk 3]
```diff
@@ -472,7 +478,12 @@ export class ChapterDetailsPage implements OnInit, OnDestroy {
         this.objRollup,
         // this.corRelationList
       );
-      this.navigateToChildrenDetailsPage(this.nextContent, 1);
+
+      const telemetryDetails = {
+        pageId: PageId.COURSE_DETAIL,
```
Comment (swayangjit) [inline src/app/curriculum-courses/chapter-details/chapter-details.page.ts:483]: PageId is incorrect.

File: src/app/collection-detail-etb/collection-detail-etb.page.ts (hunk 1 above)
Comment (NavKumarV) [reply src/app/collection-detail-etb/collection-detail-etb.page.ts:1355]: Was a mistake, will change to "COLLECTION_DETAIL"

File: src/app/curriculum-courses/chapter-details/chapter-details.page.ts (hunk 2 above)
Comment (NavKumarV) [reply src/app/curriculum-courses/chapter-details/chapter-details.page.ts:452]: Was a mistake, will change to "CHAPTER_DETAILS"

File: src/app/curriculum-courses/chapter-details/chapter-details.page.ts (hunk 3 above)
Comment (NavKumarV) [reply src/app/curriculum-courses/chapter-details/chapter-details.page.ts:483]: Was a mistake, will change to "CHAPTER_DETAILS"

**Pull request merged**